docs/eventing/accessing-traces.md

```diff
@@ -11,7 +11,7 @@ visualize and trace your requests.
 
 ## Before you begin
 
-You must have a Knative cluster running with the Eventing component installed. [Learn more](../install/README.md)
+You must have a Knative cluster running with the Eventing component installed. [Learn more](../install/_index.md)
```

Comment:
```suggestion
You must have a Knative cluster running with the Eventing component installed. [Learn more](../install/)
```

There's no need to include `_index.md` for any of these, adding the path to the directory is enough AFAIK

Reply:
fixed